core/src/main/java/org/elasticsearch/index/query/GeoBoundingBoxQueryBuilder.java

```diff
@@ -164,22 +157,9 @@ public GeoBoundingBoxQueryBuilder setCornersOGC(String bottomLeft, String topRig
         return setCornersOGC(GeoPoint.fromGeohash(bottomLeft), GeoPoint.fromGeohash(topRight));
     }
 
-    /**
-     * Specify whether or not to try and fix broken/wrapping bounding boxes.
-     * If set to true, also enables ignoreMalformed thus disabling geo point
-     * validation altogether.
-     **/
-    public GeoBoundingBoxQueryBuilder coerce(boolean coerce) {
-        if (coerce) {
-            this.ignoreMalformed = true;
-        }
-        this.coerce = coerce;
-        return this;
-    }
-
     /** Returns whether or not to try and fix broken/wrapping bounding boxes. */
     public boolean coerce() {
-        return this.coerce;
+        return this.validationMethod == GeoValidationMethod.COERCE;
     }
```

Comment:
I think we should remove this method and the ignoreMalformed() method and instead have a validationMethod() method here instead